src/common/icons/more-actions-menu-icon.tsx

```diff
@@ -10,6 +10,7 @@ export const MoreActionsMenuIcon = NamedFC('MoreActionsMenuIcon', () => (
         height="16"
         viewBox="0 0 10 16"
         fill="none"
+        stroke="currentColor"
```

Comment:
By setting `currentColor`, the SVG will inherit the color from the parent -- this works great for high contrast mode.